src/feature-extraction/features.jl

```diff
@@ -475,3 +475,23 @@ function Base.get(::Feature_Is_Colliding, rec::EntityQueueRecord{S,D,I}, roadway
     is_colliding = convert(Float64, get_first_collision(scene, vehicle_index, mem).is_colliding)
     FeatureValue(is_colliding)
 end
+
+
+#############################################
+# Compat with new interface
+#############################################
+
+generate_feature_functions("N_Lane_Right", :n_lane_right, Int, "-", lowerbound=0.0)
+function Base.get(::Feature_N_Lane_Right, scene::Frame, roadway::Roadway, vehicle_id)
+    veh = get_by_id(scene, vehicle_id)
+    nlr = get_lane(roadway, veh).tag.lane - 1
+    FeatureValue(convert(Float64, nlr))  # TODO: shouldn't this be int?
```

Comment:
The `FeatureValue` object needs `Float64` anyway. 
I think we should re-design this whole feature extraction part but it is going to be for another time. 

If you could just not use the `get` function that would be nice. There are some functions that are called `n_lanes_right` and `n_lanes_left` that should work to replace this.